### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by masinter**
Medley issue https://github.com/Interlisp/medley/issues/746

### CODE REVIEW

**Comment by nbriggs** [comment]
I think it's good -- but is there a way to see how the revised page renders before it goes live?

**Comment by masinter** [comment]
@nbriggs you can open the file locally with no server; just open file:///C:/..../Interlisp.github.io/index.html in your browser.

**Comment by nbriggs** [comment]
@masinter OK, works for me.

File: index.html [hunk 1]
```diff
@@ -106,8 +106,8 @@ <h2>Who is involved? </h2>
 
   <p>See <a href="https://docs.google.com/document/d/1cGBDNMO5yt6ymi7YiCcf6uNn6RBe5PVS7mW4kFnlzNY/edit#heading=h.z1l9hpktj4v3">Who Are We</a> section of the <a href="https://docs.google.com/document/d/1cGBDNMO5yt6ymi7YiCcf6uNn6RBe5PVS7mW4kFnlzNY/edit?usp=sharing">Annual Report</a></p>
 <p>The <a href="https://groups.google.com/g/lispcore">core group of
-contributors</a> includes <a href="https://LarryMasinter.net">Larry
-Masinter</a>, <a href="https://en.wikipedia.org/wiki/Ronald_Kaplan">Ron Kaplan</a>, Nick Briggs, Wayne Marci, Paul McJones, Bruce Mitchener, John
+contributors</a> includes <a href="https://LarryMXSasinter.net">Larry
```
Comment (stumbo) [inline index.html:109]: I think you have a typo here.

Comment (stumbo) [reply index.html:109]: A little late to the party, but beyond the one typo the changes look good to me.

**Pull request merged**